### PR TITLE
Podspec for 2.28 release with binary built on Xcode 10 Beta 6

### PR DESCRIPTION
**Pull request by AndriiMoskvin**
screen shot 2018-09-05 at 2 27 26 pm

### CODE REVIEW

File: OneMobileSDK-Xcode10b6/2.28.2631/OneMobileSDK-Xcode10b6.podspec [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  SDK provides possibility to play videos, control playback, gather metrics, play ads.
+  Checkout https://github.com/aol-public/OneMobileSDK-playground-ios for up-to-date examples.
+                       DESC
+
+  s.homepage         = 'https://github.com/vidible/OneMobileSDK-releases/releases'
+  s.author           = {
+    'Andrey Moskvin' => 'andrey.moskvin@oath.com',
+    'Alexey Demedetskiy' => 'alexey.demedetskiy@oath.com',
+    'Bogdan Bilonog' => 'bogdan.bilonog@oath.com',
+    'Roman Tysiachnik' => 'roman.tysiachnik@oath.com'
```
Comment (VladyslavAnokhin) [inline OneMobileSDK-Xcode10b6/2.28.2631/OneMobileSDK-Xcode10b6.podspec]: `Vladyslav Anokhin => vladyslav.anokhin@oath.com` =/

File: OneMobileSDK-Xcode10b6/2.28.2631/OneMobileSDK-Xcode10b6.podspec [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    text: 'Copyright 2018, Oath Inc.'
+  }
+
+  s.description = <<-DESC
+  SDK provides possibility to play videos, control playback, gather metrics, play ads.
+  Checkout https://github.com/aol-public/OneMobileSDK-playground-ios for up-to-date examples.
+                       DESC
+
+  s.homepage         = 'https://github.com/vidible/OneMobileSDK-releases/releases'
+  s.author           = {
```
[the remark this answers is not in the document]
Comment (AndriiMoskvin) [reply OneMobileSDK-Xcode10b6/2.28.2631/OneMobileSDK-Xcode10b6.podspec]: Ok, need to be fixed also in template podspecs.

File: OneMobileSDK-Xcode10b6/2.28.2631/OneMobileSDK-Xcode10b6.podspec [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    'Roman Tysiachnik' => 'roman.tysiachnik@oath.com'
+  }
+
+  s.source = {
+    http: 'https://github.com/vidible/OneMobileSDK-releases/releases/download/2.28/OneMobileSDK-Xcode10b6.framework.zip'
+  }
+
+  s.ios.deployment_target = '9.0'
+  s.tvos.deployment_target = '9.0'
+  s.requires_arc = true
```
Comment (VladyslavAnokhin) [inline OneMobileSDK-Xcode10b6/2.28.2631/OneMobileSDK-Xcode10b6.podspec]: `s.requires_arc = true`, As I remember `true` it's default value

File: OneMobileSDK-Xcode10b6/2.28.2631/OneMobileSDK-Xcode10b6.podspec (hunk 1 above)
Comment (AndriiMoskvin) [reply OneMobileSDK-Xcode10b6/2.28.2631/OneMobileSDK-Xcode10b6.podspec]: This is very critical for current Xcode 10 b 6 deliverable.

**Pull request merged**